src/main/java/wolox/training/models/Book.java

```diff
@@ -61,7 +63,7 @@
     private String isbn;
 
     @ManyToMany(mappedBy = "books")
-    private List<User> users = new ArrayList<User>();
+    private List<User> user = new ArrayList<User>();
```

Comment:
Why did you change the name of the list users to user?

Reply:
I couldn't make a attachment with the previous name and i realized that this was the problem